test/controllers/slugged_route_controller_test.exs

```diff
@@ -0,0 +1,39 @@
+defmodule CodeCorps.SluggedRouteControllerTest do
+  use CodeCorps.ConnCase
+
+  alias CodeCorps.SluggedRoute
+  alias CodeCorps.Repo
+
+  @valid_attrs %{organization_id: 42, slug: "some content", user_id: 42}
+  @invalid_attrs %{}
+
+  setup do
+    conn = %{build_conn | host: "api."}
+      |> put_req_header("accept", "application/vnd.api+json")
```

Comment:
They do `conn()` and no `host`. We need the `host` and `conn()` is deprecated in favor of `build_conn`.